mlir/test/Dialect/XeGPU/subgroup-distribute.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: mlir-opt -test-xegpu-subgroup-distribute -split-input-file %s | FileCheck %s
+
+#sg_map_16 = #xegpu.sg_map<wi_layout = [1, 16], wi_data = [1, 1]>
```

Comment:
Just a sanity check, does the distribution work fine with `wi_data` different than 1?